tests/unit/test_display.py

```
@@ -0,0 +1,347 @@
+            pr = Mock()
+            pr.number = i + 1
+            pr.title = f"PR {i + 1}"
+            pr.state = "open" if i < 2 else "closed"
+            pr.user.login = f"user{i}"
+            pr.labels = []
+            pr.created_at = datetime.now() - timedelta(days=i)
+            prs.append(pr)
+
+        pr_list = self.display.format_pr_list(prs)
+
+        for i, pr in enumerate(prs):
```

Ruff (0.13.1)

Loop control variable `i` not used within loop body

(B007)